Classes/Private/Libxml2/Converters/In/InAttributeConverter.swift

```diff
@@ -4,8 +4,8 @@ import libxml2
 extension Libxml2.In {
     class AttributeConverter: SafeConverter {
 
-        typealias Attribute = HTML.Attribute
-        typealias StringAttribute = HTML.StringAttribute
+        typealias Attribute = Libxml2.Attribute
+        typealias StringAttribute = Libxml2.StringAttribute
```

Comment:
There are lots of places where typealias's are defined for Libxml2 classes.  Maybe it would make sense to define them all once, maybe in Libxml2, and then use them through out the library?